tests/phpunit/tests/block-bindings/wpBlockBindingsRegistry.php

```diff
@@ -269,6 +269,9 @@ public function test_get_all_registered() {
 		);
 
 		$registered = $this->registry->get_all_registered();
+		$this->registry->unregister( 'test/source-one' );
```

Comment:
This one is more nuanced, but also redundant. There is a new instance of the registry created for every test using the lifecycle method `set_up`:

https://github.com/WordPress/wordpress-develop/blob/a18de617e53c0843cb3874cc4539b52296a9659b/tests/phpunit/tests/block-bindings/wpBlockBindingsRegistry.php#L33

So, the instance with registered custom sources doesn't persist between tests.

Reply:
Makes sense as well 🙂  I made this change as part of [this commit](https://github.com/WordPress/wordpress-develop/pull/7020/commits/f258a462f0969ca3a762cec096a7df96fe9d7ec1).